### PR TITLE
Delopyemts flow bug fix

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* Bug fix for refine deployments

### CODE REVIEW

File: builds/jazz-build-module/utility-loader.groovy [hunk 1]
```diff
@@ -166,4 +166,9 @@ def getApiToken(){
 	}
 }
 
+def isBuildAReplay() {
```
Comment (SiniWilson) [inline builds/jazz-build-module/utility-loader.groovy]: Please give some other name

File: builds/jazz-build-module/utility-loader.groovy [hunk 2]
```diff
@@ -166,4 +166,9 @@ def getApiToken(){
...
+  def replyClassName = "org.jenkinsci.plugins.workflow.cps.replay.ReplayCause"
```
Comment (SiniWilson) [inline builds/jazz-build-module/utility-loader.groovy]: replayClassName  instead ??

Comment (supritat) [reply builds/jazz-build-module/utility-loader.groovy]: updated

File: builds/jazz-build-module/utility-loader.groovy (hunk 1 above)
Comment (supritat) [reply builds/jazz-build-module/utility-loader.groovy]: updated

**Pull request merged**